apps/frontend/src/libs/components/app/app.tsx

```diff
@@ -27,7 +28,7 @@ const App: React.FC = () => {
 
 	return (
 		<>
-			<img alt="logo" className="App-logo" src={reactLogo} width="30" />
+			{/* <img alt="logo" className="App-logo" src={reactLogo} width="30" />
```

Comment:
You shoudn't commit comments (rule B34 of quality criteria)

Reply:
sorry, i'm forget about that